### PR TITLE
Describe how to set number of items displayed in the order summary

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes how to set number of items displayed in the order summary

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/checkout-process.html

### CODE REVIEW

File: src/sales/checkout-process.md [hunk 1]
```diff
@@ -82,6 +82,25 @@ If the company has set up [Approval Rules]({% link customers/account-dashboard-a
 ![]({% link images/images-b2b/storefront-checkout-step2-b2b.png %}){: .zoom}
 _Purchase Order Review & Payment_
 
+## Number of items displayed in the order summary
+
+The admin can change the maximum number of items displayed in the order summary at the checkout it looks neat with fewer products. By default, this value is set to 10.
```
Comment (leslietilling) [inline src/sales/checkout-process.md:87]:
```suggestion
Admin users can change the maximum number of items displayed in the order summary at checkout to streamline the display with fewer products. By default, this value is set to 10.
```

File: src/sales/checkout-process.md [hunk 2]
```diff
@@ -82,6 +82,25 @@ If the company has set up [Approval Rules]({% link customers/account-dashboard-a
...
+
+1. Go to **Stores** > **Settings** > **Configuration**.
```
Comment (leslietilling) [inline src/sales/checkout-process.md:89]:
```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
```

File: src/sales/checkout-process.md [hunk 3]
```diff
@@ -82,6 +82,25 @@ If the company has set up [Approval Rules]({% link customers/account-dashboard-a
...
+
+1. Select **Checkout** under **Sales**.
```
Comment (leslietilling) [inline src/sales/checkout-process.md:91]:
```suggestion
1. In the left panel, expand **Sales** and choose **Checkout**.
```

File: src/sales/checkout-process.md [hunk 4]
```diff
@@ -82,6 +82,25 @@ If the company has set up [Approval Rules]({% link customers/account-dashboard-a
...
+
+1. Expand the **Checkout Options** section.
```
Comment (leslietilling) [inline src/sales/checkout-process.md:93]:
```suggestion
1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Checkout Options** section.
```

File: src/sales/checkout-process.md [hunk 5]
```diff
@@ -82,6 +82,25 @@ If the company has set up [Approval Rules]({% link customers/account-dashboard-a
...
+
+1. Change the number in the field **Maximum Number of Items to Display in Order Summary**.
```
Comment (leslietilling) [inline src/sales/checkout-process.md:95]:
```suggestion
1. For **Maximum Number of Items to Display in Order Summary**, enter the maximum number of items to be displayed.
```

File: src/sales/checkout-process.md [hunk 6]
```diff
@@ -82,6 +82,25 @@ If the company has set up [Approval Rules]({% link customers/account-dashboard-a
...
+
+1. To save the configuration, click **Save Config**.
```
Comment (leslietilling) [inline src/sales/checkout-process.md:97]:
```suggestion
1. Click <span class="btn">Save Config</span>.
```

File: src/sales/checkout-process.md [hunk 7]
```diff
@@ -82,6 +82,25 @@ If the company has set up [Approval Rules]({% link customers/account-dashboard-a
...
+
+   After that, during checkout, only the specified quantity of items is displayed in the order summary:
```
Comment (leslietilling) [inline src/sales/checkout-process.md:99]:
```suggestion
   With this update, the order summary displayed during checkout is limited to the specified quantity of items.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for clarity and style

File: src/sales/checkout-process.md (hunk 7 above)
Comment (ZhovnirMaria) [reply src/sales/checkout-process.md]: Done, please check 🙏

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**